Gems/ROS2/Code/Source/RobotImporter/URDF/URDFPrefabMaker.cpp

```diff
@@ -382,7 +384,9 @@ namespace ROS2
                 linkPrefabResult.GetValue().ToString().c_str(),
                 parentEntityIter->second.GetValue().ToString().c_str());
             AZ_Trace("CreatePrefabFromUrdfOrSdf", "Link %s setting parent to %s\n", linkName.c_str(), parentName.c_str());
-            PrefabMakerUtils::SetEntityParent(linkPrefabResult.GetValue(), parentEntityIter->second.GetValue());
+            // As the link transforms are relative, use SetEntityParentRelative to make sure the link are
+            // translated, scaled and rotated to correct world location
+            PrefabMakerUtils::SetEntityParentRelative(linkPrefabResult.GetValue(), parentEntityIter->second.GetValue());
```

Comment:
This needs to be SetEntityParent to ensure that the links are set up correctly. SetEntityParentRelative is causing the mini pupper links to import in the wrong positions.

```suggestion
            PrefabMakerUtils::SetEntityParent(linkPrefabResult.GetValue(), parentEntityIter->second.GetValue());
```

Reply:
I have updated the [PrefabMakerUtils::CreateEntityUtils](https://github.com/o3de/o3de-extras/pull/591/files#diff-da2e520cc4c4bee1143c3cd7575d49ddffcd09e9a8cc1ebaa1eace954513af03R95) to create the link entities relative to their parent entity (which is the their parent SDF model) after creation to have the links be located relative to the model location using `SetEntityParentRelative`.

I have changed this code which is used to establish the parent/child hierarchy between two links joined together by an SDF joint to use `SetEntityParent` again which keeps their world transform intact.